fizzbuzz/fizzbuzz.c

```diff
@@ -19,7 +20,13 @@ int fizzbuzz(int n)
         if(i%5 == 0){
             printf("Buzz");
         }
+        if(i%5 != 0 && i%3 != 0){
+            counter++;
+        }
+        
     }
+    
+    return counter;
 }
 
```

Comment:
i think you are missing the print statement "fizzbuzz" everytime it is divisible by 15, or 3 and 5